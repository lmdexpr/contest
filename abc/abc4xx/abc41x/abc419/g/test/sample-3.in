7 18
6 7
4 5
1 7
2 7
1 4
2 5
4 6
2 3
5 6
5 7
1 5
2 4
2 6
1 2
1 3
3 4
1 6
3 5
