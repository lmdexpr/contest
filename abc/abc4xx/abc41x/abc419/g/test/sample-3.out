1 3 11 29 50 42
